st2debug/bin/submit-debug-conf.yaml

```diff
@@ -0,0 +1,12 @@
+log_file_paths:
```

Comment:
Hi Andrew, This is addressed in /packaging/root/etc/st2debug/submit-debug-info.yaml in plexxi/connect.